### PR TITLE
feat: added fw-list-options component and wiring up of fw-popover

### PR DESCRIPTION
**Pull request by kishore-kumar-E3682**
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] My commits have standard messages as mentioned in [Contributing Guidelines](./../blob/next/CONTRIBUTING.md)

### CODE REVIEW

File: packages/crayons-core/src/components/options-list/list-options.tsx [hunk 1]
```diff
@@ -55,6 +55,14 @@ export class ListOptions {
...
    */
   @Prop() isCheckbox = false;
+  /**
+   * Default option to be shown if the option doesn't match the filterText.
+   */
+  @Prop() notFoundText = 'No items Found';
+  /**
+   * Placeholder to placed on the search text box.
+   */
+  @Prop() searchText = 'Search...';
```
Comment (parsuram-vijaysankar) [inline packages/crayons-core/src/components/options-list/list-options.tsx:65]: Let's remove the "..."

**Review by parsuram-vijaysankar — APPROVED** [review]
As discussed in our review meeting let's follow up with the following items
- [ ] Dynamic search options
- [ ] Accessibility : keyboard, aria-markers

File: packages/crayons-core/src/components/options-list/list-options.tsx (hunk 1 above)
Comment (kishore-kumar-E3682) [reply packages/crayons-core/src/components/options-list/list-options.tsx:65]: The test _Search..._ is based on the DSM spec.

File: packages/crayons-core/src/components/options-list/list-options.tsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+        ? this.selectOptions.filter((option) =>
+            option.text.toLowerCase().includes(value)
+          )
+        : this.selectOptions;
+    if (this.filteredOptions.length === 0) {
+      this.filteredOptions = [{ text: 'No items Found' }];
+    }
+  }
+
+  renderSelectOption(options: Array<any>) {
```
[the remark this answers is not in the document]
Comment (kishore-kumar-E3682) [reply packages/crayons-core/src/components/options-list/list-options.tsx]: Thanks for the suggestion i have renamed it.

File: packages/crayons-core/src/components/options-list/list-options.tsx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+          disabled: option.disabled || this.value?.length >= this.max,
+        },
+      };
+    });
+    this.filteredOptions = this.selectOptions;
+  }
+
+  render() {
+    return (
+      <ul>
```
[the remark this answers is not in the document]
Comment (kishore-kumar-E3682) [reply packages/crayons-core/src/components/options-list/list-options.tsx]: Sure Parsu, Will add accessible attributes in follow up PR

**Pull request merged**